device-detector/tests/test_detector.py

```
@@ -2,6 +2,8 @@
 import pytest
 import json
 import time
+import redis
+from unittest.mock import Mock, PropertyMock, patch
```

Ruff (0.11.9)

`unittest.mock.PropertyMock` imported but unused

Remove unused import: `unittest.mock.PropertyMock`

(F401)